.github/workflows/workflow-tester.yml

```diff
@@ -34,3 +34,30 @@ jobs:
         echo ::set-output name=vault-token::$VAULT_TOKEN
         echo ::add-mask::$VAULT_TOKEN
 
+    - name: Write config
```

Comment:
I thought this was a no-op workflow. I'd expect it to print hello and stop or something. Was this block from testing?